api/api.py

```diff
@@ -1,11 +1,14 @@
 from collections import OrderedDict
+from typing import List
 import datetime
 import math
 import os
 import time
 
 from flask import Flask, request
 from flask_restful import Resource, Api
+from flask_limiter import Limiter
```

Comment:
Is `flask_limiter` a new library that has to be installed via `pip`? If so, can you add that library to `API.md` at the top, under the install instructions?

Reply:
Good catch, completely forgot to add